website/docs/registry/faq.mdx

```diff
@@ -0,0 +1,63 @@
+- **Providers**: There is no manual process for removing a version of a provider. Reach out to us via email if it is critical that a version be deleted.
+- **Policy Libraries**: The owner of a policy library can completely remove it from the Registry, through the website. [More details on this workflow can be found here](/terraform/registry/policy-libraries/publishing#removing-a-policy-library).
+
+### Can I remove a version of a provider by deleting the release from GitHub?
+
+**We recommend you never remove content from a GitHub repository that has already been published to the Terraform Registry**.
+
+The Terraform Registry acts as a kind of index for Terraform-related content that is hosted on GitHub. When you make a new tag or a new release in a GitHub repository that holds your Terraform integration, the Registry automatically creates an index record for a new content. Think of that index record as simply a pointer to the files that exist in your GitHub repository.
+
+If you remove a file, or a tag, a release, or other content from GitHub, the Registry will not be made aware that you did so. The Registry may still host index records pointing to the now-removed files or content. As a result, someone using your provider or module may see their Terraform configuration break, since through the Registry it expects to find files on GitHub which are no longer available.
+
+If you find a typo, bug, or other issue with a published version of your provider, module, or policy library, [we recommend that you fix the issue and publish a new "patch" version of the integration](#how-can-i-correct-a-bug-or-typo-in-a-version-of-my-provider-module-or-policy-library-once-it-has-been-published).
+
+### What does it mean to "resync" my provider, module, or policy library?
+
+Owners of any of these types of Terraform integrations can find a "resync" button on the webpage for the integrations they own. Triggering a resync instructs the Registry to analyze the integration's source repository on GitHub. During a resync, the registry will:
+
+- Make sure that there is a webhook established on the repository, so that new versions of the integration can be automatically published.
+- Look for any discrepancies between the versions of the integration that exist in the GitHub repository and those that exist in the Registry.
+
+If there are new versions available in GitHub that have not been published in the Registry, those new versions will be published.
+
+If there are versions that have been published in the Registry that no longer exist in GitHub, resync behavior diverges depending on the type of integration:
+
+- For modules, versions that are available in the Registry but no longer available in GitHub will be removed from the Registry. This may confuse any consumers who have come to rely on these versions of your module.
+
```

Comment:
These are some additional edge cases, that have come up as 'publishing' issues for modules. Could we highlight them with a line or two on what is the expected system behavior in such cases in terms of resync.
        Moving a Repository (to a different GH org/namespace)
	Rename module or provider